File: EIF206-Programming III/Practice/PracticaExamenI/src/main/java/Personas/Logic/Persona.java
package Personas.Logic;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;

@XmlAccessorType(XmlAccessType.FIELD)
public class Persona {
    String id;
    String nombre;
    int edad;
    String sexo;
    String estCivil;

    public Persona(String ced,String nom,int ed,String sex,String est){
        id=ced;
        nombre=nom;
        edad=ed;
        sexo=sex;
        estCivil=est;
    }
    public Persona(){this("","",0,"","");}

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public String getSexo() {
        return sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public String getEstCivil() {
        return estCivil;
    }

    public void setEstCivil(String estCivil) {
        this.estCivil = estCivil;
    }
}

File: EIF201-Programming I/Projects/Project 2/Code_Lab-Aerolinea/Persona.cpp
#include "Persona.h"



Persona::Persona(string nom, string id) {

	nombre = nom;
	this->id = id;
}

Persona::~Persona() {

}

void Persona::setNombre(string nom) {
	nombre = nom;
}
void Persona::setID(string id) {
	this->id = id;
}
string Persona::getNombre() {
	return nombre;
}
string Persona::getID() {
	return id;
}

string Persona::toString() {
	stringstream s;

	s << id << " " << nombre;

	return s.str();
}